var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -127,6 +127,8 @@ class Binutils(AutotoolsPackage, GNUMirrorPackage):
     # https://sourceware.org/bugzilla/show_bug.cgi?id=27482
     patch("parallel-build-2.36.patch", when="@2.36")
 
+    patch("macos-2.42.patch", when="@2.42 platform=darwin")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
@berquist Why the patch has been removed from the PR? :thinking:

Reply:
The patch had nothing to do with the problem; it's for compiling gold, which isn't compiled with Spack on macOS.